### PR TITLE
Load pickled fit without the model instance

### PR DESCRIPTION
**Pull request by ahartikainen**
#### Summary:

Add functionality to read old fit files, without the correct model instance.

#### Intended Effect:

Read fit, which doesn't have model instance saved.

#### How to Verify:

1. create a fit 
2. pickle only the fit
3. restart python
4. try to load pickle normally
5. load pickle with `pystan.experimental.load_fit`

tests added
~@riddell-stan do you want to add tests for experimental code too?~

#### Side Effects:

None. This is inside experimental module, which is not imported by default.

#### Documentation:

Needs update and simple example. Also needs to have disclaimer, that probably only works with the
original OS. (I have not tested with other OS, so let's test before merge).

#### Copyright and Licensing

Please list the copyright holder for the work you are submitting (this will be you or your assignee, such as a university or company):

Ari Hartikainen

By submitting this pull request, the copyright holder is agreeing to license the submitted work under the following licenses:
- Code: GPLv3 (http://opensource.org/licenses/GPL-3.0)
- Documentation: CC-BY 4.0 (https://creativecommons.org/licenses/by/4.0/)

### CODE REVIEW

File: pystan/experimental/__init__.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
-import logging
-from .misc import *
+import logging as _logging
+from .misc import fix_include
+from .pickling_tools import load_fit
 
-logger = logging.getLogger('pystan')
+_logger = _logging.getLogger('pystan')
```
Comment (riddell-stan) [inline pystan/experimental/__init__.py]: Would ``from pystan import logger`` work?

Comment (riddell-stan) [reply pystan/experimental/__init__.py]: `getLogger` is probably right thing to do. I'm just confused about why there are the added underscores. They don't seem necessary.

**Comment by riddell-stan** [comment]
Let me know when this is finalized and I'll review it more closely. In general, looks fine to me. (Although I'm frustrated, of course, that we even need this workaround.)

File: pystan/experimental/__init__.py (hunk 1 above)
Comment (ahartikainen) [reply pystan/experimental/__init__.py]: Yes, I didn't want pollute the namespace because the submodule has only 2 main functions. I can change it back.

Comment (riddell-stan) [reply pystan/experimental/__init__.py]: We don't hide `logging` anywhere else. That's what looked weird to me.

If you want to be very economical:
```
import logging
logging.getLogger('pystan').warning('...')
```

**Comment by ahartikainen** [comment]
This is ready (I rebased so tests run again).

It basically creates simple model. I used `-O1` to make compilation faster (`-O0` crashed on Windows).

It reads the module name from pickle file, but if there is a problem with this user can give it by kw.

**Comment by ahartikainen** [comment]
Oh, I still need to add example to docs

**Comment by ahartikainen** [comment]
Yes, this is ready now.

**Pull request merged**